model/DataList.php

```diff
@@ -51,6 +51,11 @@ class DataList extends ViewableData implements SS_List, SS_Filterable, SS_Sortab
 	 * @param string $dataClass - The DataObject class to query.
 	 */
 	public function __construct($dataClass) {
+		// Make sure we're getting the correct class name - this may be case-sensitive.
+		if(class_exists($dataClass)) {
+			$reflection = new ReflectionClass($dataClass);
+			$dataClass = $reflection->name;
+		}
```

Comment:
Why has this been added? Did the old manifest fix case sensitivity issues that the new version doesn’t?